ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -8,34 +8,43 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 
 expiration_date=$(date -d '4 hours' --iso=minutes --utc)
 
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
 REGION="${LEASED_RESOURCE}"
 case "${REGION}" in
 us-east-1)
-   ZONE_1=us-east-1b
-   ZONE_2=us-east-1c;;
+    ZONES=("us-east-1b" "us-east-1c")
+    ;;
+*)
+    ZONES=("${REGION}a" "${REGION}b" "${REGION}c")
 esac
-echo "AWS region: ${REGION} (zones: ${ZONE_1:-${REGION}a} ${ZONE_2:-${REGION}b})"
+
+ZONES_COUNT=${ZONES_COUNT:-2}
+ZONES=(${ZONES[@]:0:${ZONES_COUNT}})
+ZONES_STR="[ "
+ZONES_STR+=$(join_by , "${ZONES[@]}")
+ZONES_STR+=" ]"
+echo "AWS region: ${REGION} (zones: ${ZONES_STR})"
+
 
 cat >> "${CONFIG}" << EOF
 baseDomain: origin-ci-int-aws.dev.rhcloud.com
 controlPlane:
   name: master
   platform:
     aws:
-      zones:
-      - ${ZONE1:-${REGION}a}
-      - ${ZONE2:-${REGION}b}
+      zones: $ZONES_STR
 compute:
 - name: worker
   platform:
     aws:
       type: ${COMPUTE_NODE_TYPE}
-      zones:
-      - ${ZONE1:-${REGION}a}
-      - ${ZONE2:-${REGION}b}
+      zones: $ZONES_STR
 platform:
   aws:
     region: ${REGION}
     userTags:
       expirationDate: ${expiration_date}
 EOF
+
+sed -i '/\s\+$/d' ${CONFIG}
```

Comment:
What is this for?

Reply:
It was used previously to eliminate new lines but not we don't need it. Removing. Thanks for pointing it out.